app/robots/iCubGenova09/oculusConfig.ini

```diff
@@ -0,0 +1,60 @@
+name                    oculusRetargeting
+
+# ports
+leftHandPosePort        /leftHandPose:o
+rightHandPosePort       /rightHandPose:o
+playerOrientationPort   /playerOrientation:i
+rpcWalkingPort_name     /walkingRpc
+rpcVirtualizerPort_name /virtualizerRpc
+
+[GENERAL]
+samplingTime            0.01
+robot                   icub
+useXsens                1
+enableLogger            0
+enableMoveRobot         1
+# the following value is a threshold used to update the teleoperation frame position
+# when the human rotates inside the virtualizer
+playerOrientationThreshold    0.2
+
+# For kinematic scaling for task-space retargeting
+humanHeight                   1.76
+# The robot arm span is the distance between the left and right index fingertips when the arms are aligned, i.e. T pose.
+robotArmSpan                  1.0
```

Comment:
As mentioned by @S-Dafarra in f2f discussion, the values are obtained from the trials with the robot. It may need further tuning and checks in the future.